src/FileSystem/PhpFilesFinder.php

```diff
@@ -25,7 +25,7 @@ public function findInDirectoriesAndFiles(array $source): array
             }
         }
 
-        if (count($directories)) {
+        if (count($directories) > 0) {
```

Comment:
This suggested there might be minus count of directories and that it should be tested.

:-1:

Reply:
"Minous count of directories" :thinking: 

So it should be 
```php
if (count($directories) >= 1) {
```
?

Reply:
Simply

```
if (count($directories)) {

```

Reply:
But `if (count($directories)) {` is wrong as `count` returns `int`, so I'll use `if (count($directories) >= 1) {`

Reply:
That seems like super_strict_types. 

Why would you use that?